pyaedt/edb_grpc/core/edb_data/hfss_simulation_setup_data.py

```diff
@@ -0,0 +1,2146 @@
+        """Number of layer elements.
+
+        Returns
+        -------
+        str
+        """
+        return self.mesh_operation.NumLayers
+
+    @number_of_layer_elements.setter
+    def number_of_layer_elements(self, value):
+        self.mesh_operation.NumLayers = str(value)
+        self._parent._update_setup()
+
+
+class HfssPortSettings(object):
+    """Manages EDB methods for HFSS port settings."""
+
+    def __init__(self, parent):
+        self._parent = parent
+
+    @property
+    def _hfss_port_settings(self):
+        return self._parent._edb_sim_setup_info.SimulationSettings.HFSSPortSettings
+
+    @property
+    def max_delta_z0(self):
+        """Maximum change to Z0 in successive passes.
+
+        Returns
+        -------
+        float
+        """
+        return self._hfss_port_settings.MaxDeltaZ0
+
+    @max_delta_z0.setter
+    def max_delta_z0(self, value):
+        self._hfss_port_settings.MaxDeltaZ0 = value
+        self._parent._update_setup()
+
+    @property
+    def max_triangles_wave_port(self):
+        """Maximum number of triangles allowed for wave ports.
+
+        Returns
+        -------
+        int
+        """
+        return self._hfss_port_settings.MaxTrianglesWavePort
+
+    @max_triangles_wave_port.setter
+    def max_triangles_wave_port(self, value):
+        self._hfss_port_settings.MaxTrianglesWavePort = value
+        self._parent._update_setup()
+
+    @property
+    def min_triangles_wave_port(self):
+        """Minimum number of triangles allowed for wave ports.
+
+        Returns
+        -------
+        int
+        """
+        return self._hfss_port_settings.MinTrianglesWavePort
+
+    @min_triangles_wave_port.setter
+    def min_triangles_wave_port(self, value):
+        self._hfss_port_settings.MinTrianglesWavePort = value
+        self._parent._update_setup()
+
+    @property
+    def enable_set_triangles_wave_port(self):
+        """Whether to enable setting of minimum and maximum mesh limits for wave ports.
+
+        Returns
+        -------
+        bool
+            ``True`` if triangles wave port  is used, ``False`` otherwise.
+        """
+        return self._hfss_port_settings.SetTrianglesWavePort
+
+    @enable_set_triangles_wave_port.setter
+    def enable_set_triangles_wave_port(self, value):
+        self._hfss_port_settings.SetTrianglesWavePort = value
+        self._parent._update_setup()
+
+
+class HfssSolverSettings(object):
+    """Manages EDB methods for HFSS solver settings."""
+
+    def __init__(self, parent):
+        self._parent = parent
+
+    @property
+    def _hfss_solver_settings(self):
+        return self._parent._edb_sim_setup_info.SimulationSettings.HFSSSolverSettings
+
+    @property
+    def enhanced_low_freq_accuracy(self):
+        """Whether to enable legacy low-frequency sampling.
+
+        Returns
+        -------
+        bool
+            ``True`` if low frequency accuracy is used, ``False`` otherwise.
```

Comment:
```suggestion
            ``True`` if low frequency accuracy is used, ``False`` otherwise.
```
```suggestion
            ``True`` if low frequency accuracy is enabled, ``False`` otherwise.
```